code/src/layouts/best-sellers/BestSellers.tsx

```diff
@@ -9,14 +9,14 @@ import { useGetProductsQuery } from "@/store/api/productsApi";
 import "@/layouts/best-sellers/BestSellers.scss";
 
 const BestSellers = () => {
-  const { data: products, isLoading } = useGetProductsQuery();
+  const { data, isLoading } = useGetProductsQuery({ itemsPerPage: 5 });
 
   if (isLoading) return <AppTypography>Loading...</AppTypography>;
 
-  const productCards = products
-    ? products
-        .slice(0, 5)
-        .map((product) => <ProductCard key={product.id} product={product} />)
+  const productCards = data?.items
```

Comment:
I suggest using logical and instead of ternary

Reply:
Thanks, I added ternary